opennms-webapp/src/main/java/org/opennms/web/event/filter/NegativeLocationFilter.java

```diff
@@ -53,7 +53,7 @@ public String toString() {
     @Override
     public boolean equals(Object obj) {
         if (obj == null) return false;
-        if (!(obj instanceof NegativeLocationFilter)) return false;
+        if (!(obj instanceof NegativeNodeLocationFilter)) return false;
```

Comment:
I think it should be NegativeLocationFilter (like the class name), as there is another class called NegativeNodeLocationFilter